.travis.yml

```diff
@@ -1,4 +1,6 @@
 language: rust
+rust:
+  - nightly
```

Comment:
I'd really like to track stable, and that includes tests. What would be needed to run tests on stable? Where is the `stmt_expr_attributes` feature at, in terms of getting to stable? Can we avoid it?